packages/core/src/factory.ts

```diff
@@ -62,11 +63,23 @@ export function factory<Resource, Input, Output, Config extends IPrismConfig>(
     config: Config,
     validations: IPrismDiagnostic[]
   ): TE.TaskEither<Error, ResourceAndValidation & { output: Output }> => {
+    
+    const input = {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
what typescript errors are these hiding? is there another way to express what you need this to?